plugins/woocommerce-admin/client/products/sections/images-section.tsx

```diff
@@ -161,43 +174,89 @@ export const ImagesSection: React.FC = () => {
 							</CardBody>
 						) : (
 							<CardBody>
-								<MediaUploader
-									onError={ () => null }
-									onSelect={ ( file ) => {
-										if (
-											images.find(
-												( img ) => file.id === img.id
-											) === undefined
-										) {
-											setValue( 'images', [
-												...images,
-												file,
-											] );
-										}
-									} }
-									onUpload={ ( files ) => {
-										if ( files[ 0 ].id ) {
-											setValue( 'images', [
-												...images,
-												...files,
-											] );
-										}
+								<FormFileUpload
+									accept={ ALLOWED_MEDIA_TYPES }
+									multiple={ true }
+									onChange={ ( { currentTarget } ) => {
+										uploadMedia( {
+											filesList:
+												currentTarget.files as FileList,
+											onError: () => null,
+											onFileChange: onFileUpload,
+											maxUploadFileSize: 10000000,
+										} );
 									} }
-									label={
-										<>
-											<img
-												src={ DragAndDrop }
-												alt="Completed"
-												className="woocommerce-product-form__drag-and-drop-image"
+									render={ ( { openFileDialog } ) => (
+										<div
+											className="woocommerce-form-file-upload"
+											onKeyPress={ () => {} }
+											tabIndex={ 0 }
+											role="button"
+											onClick={ (
+												event: React.MouseEvent<
+													HTMLDivElement,
+													MouseEvent
+												>
+											) => {
+												const { target } = event;
+												if (
+													(
+														target as HTMLButtonElement
+													 )?.type !== 'button'
+												) {
+													openFileDialog();
+												}
+											} }
+											onBlur={ () => {} }
+										>
+											<MediaUploader
+												allowedMediaTypes={ [
+													ALLOWED_MEDIA_TYPES,
+												] }
+												onError={ () => null }
```

Comment:
Not directly related to this PR but we should probably add some error handling at some point.  I'm not sure if @mdperez86 already has an issue open for this or not as I know he implemented some error handling previously.